### ISSUE TITLE
Doctrine QueryException when starting new session

### ISSUE THREAD

**Issue by l1ppe** [Toxantron/scrumonline#120]
When deploying this project via Docker, I got the exception below after starting a new session. This also led to me not being able to enter a story or description and press start.

```
Fatal error: Uncaught Doctrine\ORM\Query\QueryException: SELECT m.id, m.name, v.value, v.highlighted FROM member m LEFT JOIN m.votes v WITH (v.member = m AND v.poll = ?1) WHERE m.session = ?2 in /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/QueryException.php on line 54

Doctrine\ORM\Query\QueryException: [Syntax Error] line 0, col 49: Error: Expected Doctrine\ORM\Query\Lexer::T_ALIASED_NAME, got 'member' in /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/QueryException.php on line 54

Call Stack:
    0.0041     407808   1. {main}() /var/www/scrumonline/src/api.php:0
    0.3385    2046904   2. PollController->current() /var/www/scrumonline/src/api.php:17
    0.4810    3813392   3. Doctrine\ORM\Query->getArrayResult() /var/www/scrumonline/src/controllers/poll-controller.php:171
    0.4810    3813392   4. Doctrine\ORM\Query->execute() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:762
    0.4810    3813392   5. Doctrine\ORM\Query->executeIgnoreQueryCache() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:947
    0.4810    3813712   6. Doctrine\ORM\Query->_doExecute() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php:992
    0.4810    3813712   7. Doctrine\ORM\Query->_parse() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php:298
    0.4972    4375824   8. Doctrine\ORM\Query\Parser->parse() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php:286
    0.4972    4375824   9. Doctrine\ORM\Query\Parser->getAST() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:360
    0.4972    4375824  10. Doctrine\ORM\Query\Parser->QueryLanguage() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:261
    0.4972    4375824  11. Doctrine\ORM\Query\Parser->SelectStatement() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:848
    0.5192    4404648  12. Doctrine\ORM\Query\Parser->FromClause() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:879
    0.5192    4404648  13. Doctrine\ORM\Query\Parser->IdentificationVariableDeclaration() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:1320
    0.5192    4404648  14. Doctrine\ORM\Query\Parser->RangeVariableDeclaration() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:1585
    0.5193    4404648  15. Doctrine\ORM\Query\Parser->AbstractSchemaName() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:1733
    0.5193    4404648  16. Doctrine\ORM\Query\Parser->match() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:975
    0.5193    4404728  17. Doctrine\ORM\Query\Parser->syntaxError() /var/www/scrumonline/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Parser.php:315
```

It seems like there is an issue with the query at line 168 of poll-controller.php, when using a Doctrine version 2.6 or higher.

I resolved this issue by setting doctrine/orm: 2.5.* in composer.json.

**Comment by Toxantron** [comment]
This might have been introduced by the `composer.lock` in #119 .

**Comment by Toxantron** [comment]
Sorry, this took so long, but I resolved this in #143. Apparently `member` became a reserved key-word somewhere down the line.
